src/languages/tptp/dolmen_tptp.ml

```diff
@@ -1,19 +1,9 @@
 
 (* This file is free software, part of dolmen. See file "LICENSE" formore information *)
 
-module type Id = Ast_tptp.Id
-module type Term = Ast_tptp.Term
-module type Statement = Ast_tptp.Statement
+(* Alias the sub-libraries *)
+module V6_3_0 = Dolmen_tptp_v6_3_0
 
-module Make
-    (L : Dolmen_intf.Location.S)
-    (I : Id)
-    (T : Term with type location := L.t and type id := I.t)
-    (S : Statement with type location := L.t and type id := I.t and type term := T.t) =
-  Dolmen_std.Transformer.Make(L)(struct
-    type token = Tokens_tptp.token
-    type statement = S.t
-    let env = ["TPTP"]
-    let incremental = true
-  end)(LexTptp)(ParseTptp.Make(L)(I)(T)(S))
+(* Include the latest version *)
+include Dolmen_tptp_v6_3_0
```

Comment:
rather than `include`, could it be `module Current = V6_3_0`? keeps things a bit tidier.

Reply:
It could, but that would break compatibility for the current users...

Reply:
ah, ok. but then I think the entrypoint for `V6_3_0` should be elsewhere so if you use the versioned API you don't see all the clutter from the current version.